cargo-auditable/build.rs

```diff
@@ -0,0 +1,107 @@
+use std::{io::prelude::*};
+use anyhow::Result;
```

Comment:
I don't think a dependency on `anyhow` is necessary here - `Box<dyn Error>` will do fine, we're not using any of `anyhow`'s advanced functionality.

Reply:
fixed